src/developer/core/plugin-api.md

```diff
@@ -78,3 +78,53 @@ from plugin import Plugin
 class MyCustomPlugin(Plugin):
     pass
 ```
+
+### Pupil Detection Plugins
+
+Pupil Core supports custom pupil detection plugins which run in the eye process. These plugins can be used to implement custom algorithms for finding the pupil position in the image, and feed them to the rest of the gaze mapping pipeline.
+
+Pupil detection plugins are supported both in Pupil Capture, as well as Pupil Player. Similar to regular user plugins, custom pupil detection plugins should be placed inside `pupil_capture_settings/plugins` for Capture, and `pupil_player_settings/plugins` for Player, both of which are located in the user directory of your system. The plugins are automatically loaded when either of the applications are started.
```

Comment:
```suggestion
Pupil detection plugins are supported in Pupil Capture, Pupil Service, and Pupil Player. Similar to regular user plugins, custom pupil detection plugins should be placed inside `pupil_capture_settings/plugins` for Pupil Capture, `pupil_service_settings/plugin` for Pupil Service, and `pupil_player_settings/plugins` for Pupil Player, both of which are located in the user directory of your system. The plugins are automatically loaded when either of the applications is started.
```

Reply:
Consider reordering to `Pupil Capture, Pupil Player, and Pupil Service`